src/openforms/js/components/admin/form_design/FormAdvancedConfiguration.js

```diff
@@ -0,0 +1,105 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import Field from 'components/admin/forms/Field';
+import Fieldset from 'components/admin/forms/Fieldset';
+import FormRow from 'components/admin/forms/FormRow';
+import {TextInput} from 'components/admin/forms/Inputs';
+import {COSIGN_OLD_TYPE_KEY} from 'components/form/coSignOld';
+
+/**
+ * Component to render the metadata admin form for an Open Forms form.
+ */
+const FormAdvancedConfiguration = ({form, formSteps, onChange}) => {
+  const {brpPersonenRequestOptions} = form;
+
+  const components = formSteps.flatMap(fs => fs.configuration.components);
```

Comment:
I think that this may miss nested components? For example components inside columns?
Maybe this util could be useful: https://github.com/open-formulieren/open-forms/blob/2bbf2a6f52cb6bb9a2d51441e5679aca83f9cd68/src/openforms/js/components/admin/form_design/utils.js#L19

Reply:
This will avoid my dirty `flatMap` as well :)